test/configs/all-customizations.json

```diff
@@ -26,12 +26,15 @@
     "customizations": {
       "hostname": "my-host",
       "kernel": {
-        "append": "debug"
+        "append": "systemd.journald.forward_to_console=1"
       },
       "user": [
         {
-          "name": "user1",
-          "key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC61wMCjOSHwbVb4VfVyl5sn497qW4PsdQ7Ty7aD6wDNZ/QjjULkDV/yW5WjDlDQ7UqFH0Sr7vywjqDizUAqK7zM5FsUKsUXWHWwg/ehKg8j9xKcMv11AkFoUoujtfAujnKODkk58XSA9whPr7qcw3vPrmog680pnMSzf9LC7J6kXfs6lkoKfBh9VnlxusCrw2yg0qI1fHAZBLPx7mW6+me71QZsS6sVz8v8KXyrXsKTdnF50FjzHcK9HXDBtSJS5wA3fkcRYymJe0o6WMWNdgSRVpoSiWaHHmFgdMUJaYoCfhXzyl7LtNb3Q+Sveg+tJK7JaRXBLMUllOlJ6ll5Hod root@localhost"
+          "name": "osbuild",
+          "key": "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNCP83Bls0eAZnMzIOTDtRFnbVZVbebfshwpilYUQ8kwLNHCiKFcdu9KjDPe0BacACP0Mna83puuH/H92BF3jlc= github.com/osbuild/images",
+          "groups": [
+            "wheel"
+          ]
```

Comment:
I currently cannot find this key shared via team BitWarden, can you add it?

Reply:
Excellent point, I revert this particular ssh key change.